packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -700,4 +700,28 @@
     <label>Rechercher à la place {{0}}</label>
     <name>quantic_SearchInsteadFor</name>
   </customLabels>
+  <customLabels>
+    <label>Non pertinent</label>
+    <name>quantic_Irrelevant</name>
+  </customLabels>
+  <customLabels>
+    <label>pas précis</label>
```

Comment:
```suggestion
    <label>Pas précis</label>
```
or even : Imprécis?

Reply:
can be yeah, will confirm with Cathrine.

Reply:
@mmitiche @SimonMilord Actually, I can't see the header for this, but if we're talking about "réponse", then it's feminine and we should make sure the options are feminine too (erronée, nocive, etc.)

Reply:
@cmartel-coveo On parle vraiment de "pas assez precis" dans ce cas-ci je crois, en fait toutes ces labels veulent dire "erronee" d'une facon ou d'une autre je crois.

I would go with "Inexact" if it was me.